mods/ts/rules/shared-vehicles.yaml

```diff
@@ -129,7 +129,7 @@ LPST:
 	Mobile:
 		Speed: 85
 		TurnSpeed: 5
-		RequiresCondition: !empdisable && undeployed
+		RequiresCondition: !empdisable && ?undeployed
```

Comment:
That is correct. In fact that is the reason I chose Python-like AND and OR operators in the first place. Implicit Boolean evaluation is clearer where the variables are treated both as Boolean values and integers.